README.md

```diff
@@ -15,33 +17,39 @@ NPM: https://www.npmjs.com/package/velocity-react
 
 **Quickstart**
 Velocity (CDN, choose one of them):  
-`<script src="cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.min.js"></script>`  
-`<script src="//cdn.jsdelivr.net/velocity/1.2.3/velocity.min.js"></script>`  
+````html
+<script src="cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.min.js"></script>
+<script src="//cdn.jsdelivr.net/velocity/1.2.3/velocity.min.js"></script>
+```
+
 Velocity UI pack (CDN, choose one of them):  
-`<script src="//cdn.jsdelivr.net/velocity/1.2.3/velocity.ui.min.js"></script>`
-`<script src="//cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.ui.min.jss"></script>`  
+````html
```

Comment:
Should be 3 back-tics not 4 for starting off this code block, or maybe there's something cool about 4 that I don't know?